featurebyte/schema/use_case.py

```diff
@@ -0,0 +1,57 @@
+"""
+Use Case API payload schema
+"""
+from typing import List, Optional
+
+from bson import ObjectId
+from pydantic import Field, StrictStr
+
+from featurebyte.models.base import FeatureByteBaseModel, PydanticObjectId
+from featurebyte.models.use_case import UseCaseModel
+from featurebyte.schema.common.base import BaseDocumentServiceUpdateSchema, PaginationMixin
+
+
+class UseCaseCreate(FeatureByteBaseModel):
+    """
+    Use Case creation schema
+    """
+
+    id: Optional[PydanticObjectId] = Field(default_factory=ObjectId, alias="_id")
+    name: StrictStr
+    context_id: PydanticObjectId
+    target_id: PydanticObjectId
+    description: Optional[StrictStr]
```

Comment:
Currently, `description` attribute can only be written during update time but not creation time. Just to remind that this slightly deviates from current pattern.

@kchua78 Will it cause any issue?